src/utils/errorTracking.ts

```diff
@@ -0,0 +1,29 @@
+import * as SentryExpo from "sentry-expo";
+
+const SENTRY_LOG_PREFIX = "[dev-sentry]";
+
+// This dev mock only implements the 4 currently used Sentry functions
+let Sentry: Pick<
+  typeof SentryExpo,
+  "init" | "setRelease" | "addBreadcrumb" | "captureException"
+>;
+
+if (__DEV__) {
+  Sentry = {
+    init: (): void => console.log(`${SENTRY_LOG_PREFIX} initialised`),
```

Comment:
Does this need to take in 1 param?

Reply:
The type is inherited from `sentry-expo`, so we don't need to worry about not specifying it here.

Since there's no important information in `init` that will be helpful when we debug, I've chosen to exclude it.